### PR TITLE
[JJBB] Increase log rotation

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Increase log rotation to help with test analysis

### CODE REVIEW

**Review by cachedout — APPROVED** [review]

**Comment by v1v** [comment]
@mdelapenya , I did add a retry for the install-dependencies, there are two approaches:
- Within the script itself -> https://github.com/elastic/e2e-testing/pull/724/commits/49c469a8eb09ac23f9647bf755db59a716e920d3
- Within the ci pipeline -> https://github.com/elastic/e2e-testing/pull/724/commits/59479d6ad26f48cdf65b770cd4dd107a16e5a1a8


I was not intended to solve it, but since It failed with the first build, I decided to add the retry, which one do you prefer?

**Review by mdelapenya — APPROVED** [review]

**Comment by mdelapenya** [comment]
> I was not intended to solve it, but since It failed with the first build, I decided to add the retry, which one do you prefer?

I prefer the second one: in the CI, as it's simpler and straight forward, and I think putting CI logic into the pipeline is clearer than in an internal script.

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -111,7 +111,7 @@ pipeline {
                   unstash 'source'
                   withGoEnv(version: "${GO_VERSION}"){
                     dir(BASE_DIR){
-                      sh script: '.ci/scripts/install-dependencies.sh', label: 'Install dependencies'
+                      retryWithSleep(retries: 2, seconds: 5, backoff: true){ sh script: '.ci/scripts/install-dependencies.sh', label: 'Install dependencies' }
```
Comment (v1v) [inline .ci/Jenkinsfile:114]: Add some CI resilience when accessing third-party systems to download dependencies are temporarily unavailable

**Comment by mdelapenya** [comment]
Failed tests is considered flaky. Restarting would fix it 🤞 ...or not... I'd merge this one ;)

**Pull request merged**